main.js

```diff
@@ -15,10 +15,11 @@
   var rangeView = document.querySelectorAll('.range-view-js');
   var updateButton = document.querySelector('#update-button');
   var inputForRange = document.querySelector('#set-range-box');
-  var randomNumber = createNumber();
   var winningSide = document.querySelector('#winning-side');
+  var gameResetButton = document.querySelector('#reset-button');
   var numberOfGuesses = 0;
-  var deleteWinnerCard
+  var deleteWinnerCard;
```

Comment:
why did you rearrange the variables like this?

Reply:
Just to have the querySelector vars together.

Reply:
Okay cool.